library/globals.inc.php

```diff
@@ -1580,14 +1580,18 @@
       '7200',                           // default
       xl('Maximum idle time in seconds before logout. Default is 7200 (2 hours).')
     ),
-
+    'secure_upload' => array(
+      xl('Secure upload files'),
+      'bool',                           // data type
+      '0',                              // default
+      xl('Block all files types that not found in the white-list. interface to edit white-list locate under the administrator->files.')
```

Comment:
for bradymiller:
will change to:
Block all files types that not found in the White List. Can find interface to edit White List at Administration->Files.